src/stores/WalletStore.ts

```diff
@@ -158,13 +155,27 @@ class WalletStore extends PersistableStore {
       void this.handleAccountChanged()
     })
     provider.on('disconnect', (error: unknown) => {
+      // Sometimes this error fires when user switching between networks too fast, we should not clean data in this case
+      if (
+        error instanceof Error &&
+        error.message.includes('Attempting to connect.')
+      ) {
+        handleError(error)
+        return
```

Comment:
Btw, you can do `return handleError(error)` as a shortcut, I think it won't create any side effects

Reply:
I guess you're right, ty